aexpect/remote.py

```diff
@@ -0,0 +1,1297 @@
+"""
+Functions and classes used for logging into guests and transferring files.
+"""
```

Comment:
One more thing, the `remote` and `rss_client` should be listed in `aexpect.__init__`.

Reply:
Good point, added in the next push. However, I will not resolve this until you confirm that the change is indeed exactly what you meant.